core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -130,56 +145,66 @@ oppia.factory('UrlInterpolationService', [
       },
 
       /**
-       * Given an resource path, returns the relative url path to that resource
-       * prefixing the appropriate cache_slug to it.
-       */
-      getStaticResourceUrl: function(resourcePath) {
-        return getCachePrefixedUrl(resourcePath);
-      },
-
-      /**
-       * Given an image path, returns the complete url path to that image
-       * prefixing the appropriate cache_slug to it.
+       * Given an image path relative to /assets/images folder,
+       * returns the complete url path to that image.
        */
       getStaticImageUrl: function(imagePath) {
         validateResourcePath(imagePath);
-        return getCachePrefixedUrl('/assets/images' + imagePath);
+        return getPrefixedUrl(
```

Comment:
Yeah, sorry I have missed it.
you mean instead of `getPrefixedUrl('/assets' + getUrlWithSlug(jsonPath));` something like `getCompleteUrl(prefix, path)`?